### PR TITLE
replace `printtable` test

### PR DESCRIPTION
**Pull request by cjprybol**
The old, now removed tests depended on dataset files we removed and was actually part of the benchmark tests. This test does the same in that it has to successfully run to pass tests, but it still doesn't check the output (same as the old tests). Is it possible to test the printed values?

### CODE REVIEW

**Comment by ararslan** [comment]
> Is it possible to test the printed values?

If it's printed using a `show` method, you can capture the string as `sprint(show, ...)`. Otherwise, you can print to a temp file then `readchomp` the contents.

**Comment by cjprybol** [comment]
Thanks for that suggestion @ararslan, worked great!

**Review by nalimilan — APPROVED** [review]
Thanks.

**Comment by nalimilan** [comment]
For future reference, the `printtable` tests were mistakenly removed by #26.

File: test/io.jl [hunk 1]
```diff
@@ -38,4 +40,14 @@ module TestIO
     show(io, "text/html", dt)
     @test length(String(take!(io))) < 10000
 
+    dt = DataTable(A = 1:26,
+                   B = 'a':'z',
+                   C = [string(x) for x='A':'Z'],
+                   D = CategoricalArray([string(x) for x='A':'Z']),
+                   E = NullableArray(rand(26)),
```
Comment (cjprybol) [inline test/io.jl:47]: I can't believe I put rand here...

**Pull request merged**